components/object-property-list/object-property-list-item.js

```diff
@@ -44,28 +45,33 @@ export class ObjectPropertyListItem extends RtlMixin(LitElement) {
 			:host([dir="rtl"]) .item-icon {
 				margin: -0.1rem 0 0 0.3rem;
 			}
+			:host([skeleton]) .d2l-skeletize {
+				display: inline-flex;
+				height: 0.7rem;
+				width: 100px;
+			}
```

Comment:
I'm still thinking we should be leaving these spans un-modified (inline display) and expecting placeholder text.

Reply:
@dlockhart You were mentioning approaches like `d2l-skeletize-<width>`, is it standard practice to use empty fixed-width divs for skeletons, or to use placeholder text?

Reply:
Code searching around, I'm seeing quite a few `<div class="d2l-skeletize-75">${someValueThatWillEventuallyBeThere}</div>`.

It's harder to search for placeholder text. Placeholder text is also interesting in that it probably needs to be localized (even though it's not shown!) and would show up for screen readers which might be a strange experience.

Reply:
Yeah exactly, the screen reader experience is what I was talking about here:

> Not sure what's best for accessibility here, whether we'd want a descriptive placeholder or for it to read as nothing...

Could it also be a problem if it had no screen-reader content while skeletized? Would we want users to know that there's content coming? Should that content get announced when it becomes available?

According to [this doc](http://proddevutil.desire2learn.d2l/d2c/content/product%20design/design.d2l%20archive/Skeleton%20Page%20Loaders%20%E2%80%93%20Daylight%20Design%20System.html), it sounds like we should be setting `aria-busy="true"` on the parent element while there's a skeleton...

Reply:
(This is why I'm not putting a ton of weight in that doc. Colin wrote it well before we ended up implementing it, so it didn't get a technical review. I don't even think we used it during implementation.)

Reply:
I was playing around with this a bit on Friday.

I agree with @eKoopmans that we should maintain the display: inline. When I was playing around with it, I was getting pretty good layout when the dot separators showing, but I was using some placeholder text... and I was thinking we might have to add `aria-hidden: true` while in skeleton mode.

Reply:
One of the nice things about having some placeholder text, is that we get the height set by the browser in an equivalent way. No changing to inline-block necessary to apply a one-off height to _try_ and match layout. Also, no issues trying to match the baseline.

Reply:
I can try to change the span to be unmodified.

If we want to go with the placeholder text, any thoughts on what it should be?

Reply:
I'd say the consumer should specify the placeholder text. In the case of the skeleton at the list level itself - maybe `Item 1` etc?

And yep maybe best just to stick `aria-hidden="true"` on all skeleton items...